ci/upload_to_owncloud.sh

```diff
@@ -50,6 +54,17 @@ main() {
     remote_path="$1"; shift
     [ -n "$1" ] || { usage; err "Missing local-path"; exit 1; }
     local_path="$1"; shift
+    [ -z "${REMOTE_TEMP_PATH-unset}" ] && { usage; err "Temporary remote path is set but empty!"; exit 1; }
+
+    if [ -n "$REMOTE_TEMP_PATH" ] ; then
+        UPLOAD_URL="$OWNCLOUD_URL/$user/$REMOTE_TEMP_PATH"
```

Comment:
I think we should put the temp stuff together in one directory, so we can easily clean it up if anything is left behind from crashed scripts:
```suggestion
        UPLOAD_URL="$OWNCLOUD_URL/$user/temp_upload/$REMOTE_TEMP_PATH"
```